content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -353,6 +353,36 @@ spec:
 {{< /tabs >}}
 
 
+### Version removal
+
+An API version can be removed if there is a newer API version served and stored in the API server for at least 3 Kubernetes releases.
```

Comment:
There's nothing inherent about 3 releases before removing a version, that's just how the storage API chose to manage the version

Reply:
Removed this line.